tools/apkdiff/apkdiff.csproj

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Now I think about it, should all new projects be SDK style?

Reply:
Makes sense. Do we have a migration tool available on Mac or do I need to do it by hand?

Reply:
@radekdoulik I found they are so short you can type them now:
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net472</TargetFramework>
  </PropertyGroup>
</Project>
```
It has a bunch of conventions based on the `csproj` file name. You can also delete `Properties\AssemblyInfo.cs`.

Reply:
Thanks! I did similar thing here https://github.com/radekdoulik/apkdiff/commit/8608ae3085ae7ccdab85735f3e84e270557f3c37

I guess I will need to modify the `OutputPath` in addition to that in XA.